package.json

```diff
@@ -25,8 +25,13 @@
     "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test"
   },
+  "overrides": {
+    "@lblod/ember-environment-banner": {
+      "ember-source": "$ember-source"
```

Comment:
This is needed because the addon only lists v4 as a valid peerDep, but v5 works as well.